BoxContentSDK/BoxContentSDK/Requests/BOXRequest_Private.h

```diff
@@ -71,6 +72,12 @@
                                  successBlock:(BOXAPIHeaderSuccessBlock)successBlock
                                  failureBlock:(BOXAPIHeaderFailureBlock)failureBlock;
 
+- (BOXAPIOAuth2ToJSONOperation *)authOperationWithURL:(NSURL *)URL
```

Comment:
If we're including in the contentSDK, does 'private' make sense? Maybe BOXRequest+Authentication?